CppExport/src/visitors/header_visitors/ExpressionVisitorHeader.cpp

```diff
@@ -194,10 +196,22 @@ SourceFragment* ExpressionVisitorHeader::visit(Expression* expression)
 			case UnaryOperation::PARENTHESIS: *fragment << "(" << visit(e->operand()) << ")"; break;
 			case UnaryOperation::DEREFERENCE: *fragment << "*" << visit(e->operand()); break;
 			case UnaryOperation::ADDRESSOF: *fragment << "&" << visit(e->operand()); break;
-			default: error(e, "Unkown unary operator type");
+			default: error(e, "Unknown unary operator type");
 		}
 	}
-	else if (auto e = DCast<TypeTraitExpression>(expression)) notAllowed(e);
+	else if (auto e = DCast<TypeTraitExpression>(expression))
+	{
+		if (e->typeTraitKind() == TypeTraitExpression::TypeTraitKind::SizeOf)
+			*fragment << "sizeof";
```

Comment:
Please put all three of these on the previous line do the same for the source visitor.